cypress/integration/rich-text/RichTextEditor.spec.ts

```diff
@@ -359,6 +359,63 @@ describe('Rich Text Editor', { viewportHeight: 2000 }, () => {
       });
     });
 
+    it('should be deleted if empty when pressing delete', () => {
+      headings.forEach(([type]) => {
+        const value = 'some text';
+        richText.editor.click().type(value);
+
+        clickDropdownItem(type);
+
+        cy.findByTestId('toolbar-entity-dropdown-toggle').click();
+        cy.findByTestId('toolbar-toggle-embedded-entry-block').click();
+
+        richText.editor.type('{uparrow}').type(
+          Array(value.length)
+            .fill('')
+            .map(() => '{del}')
+            .concat('{del}')
+            .join('')
+        );
+
+        richText.expectValue(
+          doc(
+            block(BLOCKS.EMBEDDED_ENTRY, {
+              target: {
+                sys: {
+                  id: 'example-entity-id',
+                  type: 'Link',
+                  linkType: 'Entry',
+                },
+              },
+            }),
```

Comment:
Should we add tests for other blocks such as lists, blockquotes, hr, etc or that's enought?